internal/cmd/exodus.go

```diff
@@ -193,14 +193,14 @@ func exodusMain(ctx context.Context, cfg conf.Config, args args.Config) int {
 			"MIME type", mtype.String(),
 			"error", err,
 		).Debug("MIME type detection attempted")
-		gwItem.ContentType = mtype.String()
 
 		if item.LinkTo != "" {
 			linkSrcDirRelative := path.Dir(getRelPath(item.SrcPath, args.Src))
 			linkSrcDirFull := path.Join(destTree, linkSrcDirRelative)
 			gwItem.LinkTo = path.Join(linkSrcDirFull, "/", item.LinkTo)
 		} else {
 			gwItem.ObjectKey = item.Key
+			gwItem.ContentType = mtype.String()
```

Comment:
This should work, but would it make more sense to move the entire mtype calculation into this block?  Otherwise we waste time calculating the content type (which requires reading the file) even if we don't use it.  Also if SrcPath is a broken symlink, which it often will be, I guess the "MIME type detection attempted" log message will show an error which might be confusing during troubleshooting.